index.html

```diff
@@ -303,30 +311,24 @@
       <h2>Introduction</h2>
 
       <p>
-[=Credentials=] are integral to our daily lives; driver's licenses confirm
+[=Credentials=] are integral to our daily lives: driver's licenses confirm
 our capability to operate motor vehicles, university degrees assert our level
-of education, and government-issued passports permit travel between countries.
-This specification provides a mechanism to express these sorts of
-[=credentials=] on the Web in cryptographically secure, privacy-respecting,
-and machine-verifiable way. These [=credentials=] provide
-benefits to us when used in the physical world, but their use on the Web
-continues to be elusive.
+of education, and government-issued passports attest to our citizenship when 
+traveling between countries. This specification provides a mechanism for
+expressing these sorts of [=credentials=] on the Web in a way that is 
+cryptographically secure, privacy respecting, and machine verifiable. These
+[=credentials=] provide benefits to us when used in the physical world, but
+their use on the Web continues to be elusive.
```

Comment:
Must be applied with preceding change request
```suggestion
of education; and government-issued passports attest to our citizenship when 
traveling between countries. This specification provides a mechanism for
expressing these sorts of [=credentials=] on the Web in a way that is 
cryptographically secure, privacy respecting, and machine verifiable. These
[=credentials=] provide benefits to us when used in the physical world, but
their use on the Web continues to be elusive.
```